examples/glade-cmip6.json

```
@@ -0,0 +1,137 @@
+{
+  "stac_version": "0.9.0",
```

STAC 1.0.0-beta.1 has been released.
```suggestion
  "stac_version": "1.0.0-beta.1",
```